mix.exs

```diff
@@ -31,7 +70,7 @@ defmodule KinesisClient.Mixfile do
   # Run "mix help compile.app" to learn about applications.
   def application do
     [
-      extra_applications: [:logger]
+      extra_applications: [:logger, :wx, :observer, :runtime_tools]
```

Comment:
Do we want to remove the wx and observer as application dependencies before merging? 😅

Reply:
similar to above, this is a dev only thing and will be removed later